package proto_ff;

import "yd_fieldoptions.proto";


message areaarea
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "区域ID"];
	optional int32 subType = 2[(yd_fieldoptions.field_cname) = "细分类型"];
	optional int64 belongToSceneID = 3[(yd_fieldoptions.field_cname) = "所属场景"];
	optional int32 shapeType = 4[(yd_fieldoptions.field_cname) = "形状类型"];
	optional int32 isNotice = 5[(yd_fieldoptions.field_cname) = "是否进出提示"];
}


message Sheet_areaarea
{
	repeated areaarea areaarea_List = 1[(yd_fieldoptions.field_arysize)=300];
}

message areapath
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "路径ID"];
	optional int64 belongToSceneID = 2[(yd_fieldoptions.field_cname) = "所属场景"];
}


message Sheet_areapath
{
	repeated areapath areapath_List = 1[(yd_fieldoptions.field_arysize)=2000];
}
